db/migrate/20230406211649_create_authentication_events.rb

```diff
@@ -0,0 +1,11 @@
+class CreateAuthenticationEvents < ActiveRecord::Migration[6.0]
+  def change
+    create_table :authentication_events do |t|
```

Comment:
The migration file should include an index on the `email_submitted` and `event_type` columns to improve query performance, especially if the `authentication_events` table grows large.